ingestion-edge/ingestion_edge/disk_queue.py

```diff
@@ -0,0 +1,70 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+from .conf import QUEUE_FILE
+from .conf import QUEUE_POSITION_FILE
+from flask import Blueprint
+from google.cloud.pubsub import PublisherClient
+from os import remove
+from os.path import exists
```

Comment:
I tend to prefer importing `os` and `os.path` as packages and then referencing these functions with fully qualified names (os.remove, os.path.exists) since the function names themselves are so short. In particular, I had assumed the call to `remove` below was some locally defined function.

Similarly, this would read more clearly to me if we used `threading.Lock()` below.